lib/api.js

```diff
@@ -16,13 +15,15 @@ if (config.env === 'development') {
   const build = require('mastarm/lib/build')
   const util = require('mastarm/lib/util')
 
-  build({
-    config: config._defaultConfig,
-    files: util.parseEntries(config.entries),
-    watch: true
-  })
-  .then(() => console.log('JavaScript & CSS built..'))
-  .catch((err) => console.error('Error building JavaScript & CSS...', err))
+  if (process.argv.indexOf('no-build') === -1) {
```

Comment:
I was making lots of changes to the server and whenever anything changed, the server would restart and cause a rebuild of all of the js.  I added this option so that it wouldn't rebuild the js.